### DISCUSSION TITLE
Package importing method in quasar

### DISCUSSION THREAD

**Discussion by narendraparihar** [quasarframework/quasar#13593] [CLI - PWA mode]
I am facing problems while importing certain packages. Is there any particular method to import a package rather than  'vue.component' 
and 'vue.use' for more specifications about a question  I am  unable to import 'vue-horizontal-list' package in my project it shows below error :
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '_c')

**Comment by MilosPaunovic** [comment]
If this is the package you're trying to import [https://github.com/fuxingloh/vue-horizontal-list](https://github.com/fuxingloh/vue-horizontal-list), it seems like it's dependency is Vue@2, and Quasar@2 is built on top of Vue@3.
